dot/core/service_integration_test.go

```diff
@@ -52,7 +51,7 @@ func balanceKey(t *testing.T, pub []byte) (bKey []byte) {
 
 func generateTestValidRemarkTxns(t *testing.T, pubKey []byte, accInfo types.AccountInfo) ([]byte, runtime.Instance) {
 	t.Helper()
-	projectRootPath := filepath.Join(utils.GetProjectRootPathTest(t), "chain/gssmr/genesis.json")
+	projectRootPath := utils.GetGssmrV3SubstrateGenesisRawPathTest(t)
```

Comment:
Nice, first thing I looked for in potential conflicts of my branch was hard coded paths